src/utils.js

```diff
@@ -1,5 +1,5 @@
 import redent from 'redent'
-import cssParse from 'css/lib/parse'
```

Comment:
Directly loading `dist/cjs/cssTools` feels somewhat hacky as there is no guarantee this path will exist in future releases of `@adobe/css-tools`. I wonder if using `require` instead would work.

Reply:
I agree but I can't found a way to load the CJS build.

If I do without a path:
```
    ReferenceError: self is not defined

      1 | import redent from 'redent'
    > 2 | import {parse} from '@adobe/css-tools'
        | ^
```
If I do a require:
```
   ReferenceError: self is not defined

      1 | import redent from 'redent'
      2 | import isEqual from 'lodash/isEqual'
    > 3 | const {parse} = require('@adobe/css-tools')
        |                 ^
```

Let me know if you have a better solution or something that I could change for the build of `@adobe/css-tools`

Reply:
I made a change of how `@adobe/css-parse` is build (UMD build, globalThis is now set to this in wepack config). This should solve the issue/concern.